.github/workflows/style-check.yaml

```diff
@@ -18,8 +18,9 @@ jobs:
 
       - name: Clang Format
         run: |
-          brew install clang-format
-          PATH="$(brew --prefix clang-format)/bin:${PATH}"
+          pip install --upgrade clang-format
+          which clang-format
```

Comment:
Why do we need this?

Reply:
If this job fails this and the clang-format version are the first things you're looking for if you cannot reproduce the failure locally.